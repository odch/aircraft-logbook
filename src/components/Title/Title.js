import React from 'react'
import PropTypes from 'prop-types'
import Typography from '@material-ui/core/Typography'

const Title = props => (
  <Typography variant="title" align="center">
    {props.text}
  </Typography>
)

Title.propTypes = {
  text: PropTypes.string.isRequired
}

export default Title
