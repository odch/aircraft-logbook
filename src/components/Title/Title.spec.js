import React from 'react'
import renderer from 'react-test-renderer'
import Title from './Title'

describe('components', () => {
  describe('Title', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<Title text="test title" />).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
