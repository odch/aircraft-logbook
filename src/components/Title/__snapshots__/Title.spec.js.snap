// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Title renders correctly 1`] = `
<h2
  className="MuiTypography-root-1 MuiTypography-title-7 MuiTypography-alignCenter-15"
>
  test title
</h2>
`;
